buildSrc/build.gradle.kts

```diff
@@ -1,11 +1,32 @@
 plugins {
+    idea
     `kotlin-dsl`
+    kotlin("jvm") version "1.6.21"
 }
 
-repositories {
-    mavenCentral()
-}
+// set the versions of Gradle plugins that the subprojects will use here
```

Comment:
Previously the plugin versions were set in the root `settings.gradle.kts`. Now they should be set here.